### DISCUSSION TITLE
Cannot select foreign key on O2M relation

### DISCUSSION THREAD

**Discussion by ChappIO** [directus/directus#17537] [Archive]
I created a collection with a M2O relation, now on the opposite site I would like to add a O2M relation for viewing. But for some reason I cannot continue. What might be the reason for this?

![image](https://user-images.githubusercontent.com/13708940/219067695-b5aa8388-1b87-4808-bdd8-645af9896fda.png)

**Comment by rijkvanzanten** [comment]
This is the reason: https://github.com/directus/directus/issues/16692 🙂
